be/src/storage/lake/starlet_group_assigner.cpp

```diff
@@ -0,0 +1,20 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2021-present, StarRocks Limited.
+
+#include "storage/lake/starlet_group_assigner.h"
+
+#ifdef USE_STAROS
+
+#include "service/staros_worker.h"
+
+namespace starrocks::lake {
+
+StatusOr<std::string> StarletGroupAssigner::get_group(int64_t tablet_id) {
+    return get_staros_shard_path(tablet_id);
```

Comment:
The group path is a subdirectory of  the`object_storage_uri` stored in ShardInfo. group path should be constructed from `object_storage_uri` and a custom property stored in `ShardInfo` by FE, like this:
```
    RETURN_IF_ERROR(auto shard_info, get_shard_info(tablet_id));
    auto group_id = shard_info.get_property("GroupID");
    return fmt::format("{}/{}", shard_info.object_storage_uri, group_id);
```
Since the Starlet does not provide any API for getting custom properties, you can
use a fixed `group_id` now, like this:
```
    RETURN_IF_ERROR(auto shard_info, get_shard_info(tablet_id));
    auto group_id = 1001; // TODO: get group id from ShardInfo
    return fmt::format("{}/{}", shard_info.object_storage_uri, group_id);
```